solutions/swb-app/src/errors/databaseError.ts

```diff
@@ -3,6 +3,7 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
+// TODO: is this error needed after sshKeyService is done?
```

Comment:
Nit: delete these TODOs and create a SIM to re-evaluate if these errors are necessary.

Reply:
Made GALI-2246 for these and will take the TODOs out in the next PR